server/channels/app/imports/import_validators_test.go

```diff
@@ -1023,30 +1023,47 @@ func TestImportValidatePostImportData(t *testing.T) {
 func TestImportValidateDirectChannelImportData(t *testing.T) {
 	// Test with valid number of members for direct message.
 	data := DirectChannelImportData{
-		Members: &[]string{
-			model.NewId(),
-			model.NewId(),
+		Participants: []*DirectChannelMemberImportData{
+			{
+				Username: ptrStr(model.NewId()),
+			},
+			{
+				Username: ptrStr(model.NewId()),
+			},
+			{
+				Username: ptrStr(model.NewId()),
+			},
 		},
 	}
 	err := ValidateDirectChannelImportData(&data)
 	require.Nil(t, err, "Validation failed but should have been valid.")
 
 	// Test with valid number of members for group message.
 	data = DirectChannelImportData{
-		Members: &[]string{
-			model.NewId(),
-			model.NewId(),
-			model.NewId(),
+		Participants: []*DirectChannelMemberImportData{
+			{
+				Username: ptrStr(model.NewId()),
+			},
+			{
+				Username: ptrStr(model.NewId()),
+			},
+			{
+				Username: ptrStr(model.NewId()),
+			},
 		},
 	}
 	err = ValidateDirectChannelImportData(&data)
 	require.Nil(t, err, "Validation failed but should have been valid.")
 
 	// Test with all the combinations of optional parameters.
 	data = DirectChannelImportData{
-		Members: &[]string{
-			model.NewId(),
-			model.NewId(),
+		Participants: []*DirectChannelMemberImportData{
+			{
+				Username: ptrStr(model.NewId()),
```

Comment:
0/5 minor nit: Now you can use `model.NewPointer` for anything that you want to get a pointer of. I'd like to get rid of `ptrStr`. But can be done later as well. Up to you.